common/testData/transforms/byn-Latn-t-byn-ethi-m0-xaleget.txt

```diff
@@ -1,1091 +1,1091 @@
-ሓለይና	xaleyna
-ሓለፍና	xalefna
-ሓላል	xalal
-ሓላል	xalal
-ሓላል	xalal
-ሓላም	xalam
-ጠላም	thelam
-ሓልሓል/62’	xalxal/62’
-ሓልብና	xalbna
-ሓልቲ	xalti
-ሓሎ	xalo
-ሓሓይ	xaxay
-ሓሕ	xax
-ሓሕ	xax
-ሓመልሚል	xamelmil
-ሓምደ	xamde
-ሓመድ	xamed
-ሓመ’ድና	xame’dna
-ሓሙላ	xamula
-ሓሚስ	xamis
-ሓማሴን	xamasén
-ሓማስ	xamas
-ሓምል	xaml
-ሓምል	xaml
-ሓምሓም	xamxam
-ሓምበላይ	xambelay
-ሓምቦ	xambo
-ሓምተራ	xamtera
-ሓምታ	xamta
-ሓምዳ	xamda
-ሓምዳ	xamda
-ሓሰልና	xaselna
-ሓሰስና	xasesna
-ሓሰስና	xasesna
-ሓሰብና	xasebna
-ሓሰይና	xaseyna
-ሓስተና	xastena
-ሓሶሰይና	xasoseyna
-ሓረስና	xaresna
-ሓረስድና	xaresdna
-ሓረርና	xarerna
-ሓረትስና	xaretsna
-ሓረብና	xarebna
-ሓረጥስና	xarethsna
-ሓረጥና	xarethna
-ሓረጥና	xarethna
-ሓረፍና	xarefna
-ሓሩቀት	xaruqet
-ሓሪር	xarir
-ሓራሙዅ	xaramukhwu
-ሓራም	xaram
-ሓራሲ	xarasi
-ሓራቅርና	xaraqrna
-ሓርሓራ	xarxara
-ሓርሩም	xarrum
-ሓርብ	xarb
-ሓርኮክያ	xarkokya
-ሓርጨምድና	xarchemdna
-ሓሸላ	xashela
-ሓሸር	xasher
-ሓሻእ	xashae
-ሓሻኪቶ	xashakito
-ሓቀፍና	xaqefna
-ሓቀፍድና	xaqefdna
-ሓቓቕና	xaqhaqhna
-ሓቓቕና	xaqhaqhna
-ሓቕ	xaqh
-ሓቕ	xaqh
-ሓበስና	xabesna
-ሓበስትና	xabestna
-ሓበርስና	xabersna
-ሓበርና	xaberna
-ሓበሽዋር	xabeshwar
-ሓበናታዅ	xabenatakhwu
-ሓበናት	xabenat
-ሓበን	xaben
-ሓበንድና	xabendna
-ሓበካ	xabeka
-ሓቢና	xabina
-ሓቢና	xabina
-ሓባሊኮራ	xabalikora
-ሓባብ	xabab
-ሓብሓብና	xabxabna
-ሓብለክና	xablekna
-ሓብለክና	xablekna
-ሓብሳ	xabsa
-ሓንደካ	xandeka
-ሓንደይና	xandeyna
-ሓንፈሲስና	xanfesisna
-ሓንፈስና	xanfesna
-ሓንጐል	xangwel
-ሓንጐጥና	xangwethna
-ሓንኰልና	xankwelna
-ሓንኰኵና	xankwekwuna
-ሓንቍልና	xanqwulna
-ሓኮራ	xakora
-ሓኽ	xakh
-ሓወሽ	xawesh
-ሓወና	xawena
-ሓዋን	xawan
-ሓወጭና	xawechna
-ሓዋርያ	xawarya
-ሓዋይት	xawayt
-ሓውስና	xawsna
-ሓው	xaw
-ሓ’ው	xa’w
-ሓየ	xaye
-ሓየልና	xayelna
-ሓየምና	xayemna
-ሓያም	xayam
-ሓዮ¨’83	xayo¨’83
-ሓይሉ	xaylu
-ሓይልንኬል	xaylnkél
-ሓደምበስ	xadembes
-ሓደምበስቍር	xadembesqwur
-ሓደስና	xadesna
-ሓደርና	xaderna
-ሓደና	xadena
-ሓደን	xaden
-ሓዲስ	xadis
-ሓዳይ	xaday
-ሓዲስ	xadis
-ሓጅራ	xajra
-ሓገስትና	xagestna
-ሓገድና	xagedna
-ሓገይና	xageyna
-ሓጋይ	xagay
-ሓጋግ	xagag
-ሓጎቲባ	xagotiba
-ሓጠርና	xatherna
-ሓጠይ	xathey
-ሓጢኣት	xathiat
-ሓጢኣቱዅ	xathiatukhwu
-ሓጣእ	xathae
-ሓጥር	xathr
-ሓጥርና	xathrna
-ሓጥሺ	xathshi
-ሓጥሺ	xathshi
-ሓጥእና	xathena
-ሓጨይስና	xacheysna
-ሓጨይና	xacheyna
-ሓጨፉራ	xachefura
-ሓጨፍና	xachefna
-ሓጩፍ	xachuf
-ሓጭፍ	xachf
-ሓጭፍ	xachf
-ሓፈድና	xafedna
-ሓፈፍሎራ	xafeflora
-ሓፈፍና	xafefna
-ሓፉላዅ	xafulakhwu
-ሓፋር	xafar
-ሓፋር	xafar
-ሓፍሓፍና	xafxafna
-ሓዘና	xazena
-ሓዘን	xazen
-ሓጐጕና	xagwegwuna
-ሔማ	xéma
-ሕላ	xla
-ሕላብደራ	xlabdera
-ሕላይ	xlay
-ሕልባ	xlba
-ሕልቁም	xlqum
-ሕሒሕ	xxix
-ሕማር	xmar
-ሕምሓም	xmxam
-ሕምሕምና	xmxmna
-ሕምሕምድና	xmxmdna
-ሕምስ	xms
-ሕሳብ	xsab
-ሕሩጭ	xruch
-ሕራብ	xrab
-ሕርም	xrm
-ሕርበት	xrbet
-ሕርጥ	xrth
-ሕሮራ	xrora
-ሕሹኵሹኵ	xshukwushukwu
-ሕሹኵሹኵ	xshukwushukwu
-ሕሻኪ	xshaki
-ሕሽም	xshm
-ሕሽኵል	xshkwul
-ሕቕ	xqh
-ሕበራ	xbera
-ሕበርበሮ	xberbero
-ሕበበርና	xbeberna
-ሕቡብ	xbub
-ሕብሕባ	xbxba
-ሕብር	xbr
-ሕብጥ	xbth
-ሕትካ	xtka
-ሕንሽርሽሩ	xnshrshru
-ሕያት	xyat
-ሕንደራ	xndera
-ሕንዳቓ	xndaqha
-ሕንደራ	xndera
-ሕንዶሪራ	xndorira
-ሕንጁራ	xnjura
-ሕንጊግ	xngig
-ሕንጉላ	xngula
-ሕንጣናር	xnthanar
-ሕንጣንትና	xnthantna
-ሕንጣናዅ	xnthanakhwu
-ሕንጩና	xnchuna
-ሕንጪናቅ	xnchinaq
-ሕንጫ	xncha
-ሕንፍልና	xnflna
-ሕንቍልና	xnqwulna
-ሕኖት	xnot
-ሕኖት	xnot
-ሕወር	xwer
-ሕውን	xwn
-ሕደራ	xdera
-ሕዳም	xdam
-ሕዳረብ	xdareb
-ሕዳር	xdar
-ሕድኣት	xdat
-ሕጅቅ	xjq
-ሕጣን	xthan
-ሕጊ	xgi
-ሕጩይ	xchuy
-ሕጪት	xchit
-ሕጩፍ	xchuf
-ሕፍት	xft
-ሖጀብ	xojeb
-ሖግ	xog
+ሁለቱም	huletum
+ሁሉ	hulu
+ሁሉም	hulum
+ሁኔታ	huneeta

```

Comment:
@dyacob the change from hunéta to huneeta could use some clarification. Per the ticket, "Xalaget" = "Transliteration from the Latin convention as defined by the government of Eritrea for education in the Blin language". Is there a reference for Xalaget that clarifies hunéta vs huneeta? And why this file had hunéta previously?

Reply:
@btangmu I was surprised by this and had to wonder about it also. A limitation was that the information that I had to work with did not come directly from the Eritrean Ministry of Education, but from secondary sources. They in turn did not address vowels clearly enough, or I may have misunderstood.  One source clearly was in error, indicating that "ie" should be used.

Converting a full book that came from the education ministry brought the problem to light.  I've attached the introduction to the book here as a reference.
  
[BlinChildrensBook-Intro.pdf](https://github.com/unicode-org/cldr/files/11255006/BlinChildrensBook-Intro.pdf)